roles/dds_cli/defaults/main.yml

```diff
@@ -1,2 +1,2 @@
 ---
-dds_cli_version: 0.0.11
+dds_cli_version: 1.0.0
```

Comment:
The latest is 1.0.3 now https://github.com/ScilifelabDataCentre/dds_cli/releases

Reply:
Fixed